pkg/minikube/download/preload.go

```diff
@@ -141,6 +142,10 @@ func Preload(k8sVersion, containerRuntime string) error {
 	out.Step(style.FileDownload, "Downloading Kubernetes {{.version}} preload ...", out.V{"version": k8sVersion})
 	url := remoteTarballURL(k8sVersion, containerRuntime)
 
+	if checksum, err := getChecksum(k8sVersion, containerRuntime); err == nil {
+		url += "?checksum=" + checksum
+	}
```

Comment:
how about adding a warning log when there is err is not nil and just log it for reference ?

Reply:
we already break when failing to save the checksum, so might as well do the same here